vibes/vibed-ui/src/ui/Theme.cpp

```
@@ -0,0 +1,190 @@
+    std::size_t numberEnd = numberBegin;
+    if (json[numberEnd] == '-') {
+        ++numberEnd;
+    }
+    while (numberEnd < json.size() && std::isdigit(static_cast<unsigned char>(json[numberEnd])) != 0) {
+        ++numberEnd;
+    }
+    if (numberEnd <= numberBegin) {
+        return false;
+    }
+
+    const std::string numberText = json.substr(numberBegin, numberEnd - numberBegin);
```

Clang (14.0.6)

[warning] variable 'numberText' is not initialized

(cppcoreguidelines-init-variables)